flickr_download/flick_download.py

```diff
@@ -317,6 +317,25 @@ def serialize_json(obj):
     return ret
 
 
+def get_photo_sizes(photo):
+    pSize = {}
+    for attempt in range(1, 5):
+        try:
+            pSize = photo.getSizes()
+        except FlickrError as ex:
+            logging.warning("Flickr error getting photo size: {}, "
+                            "attempt: #{}, "
+                            "attempts left: {}".format(str(ex), attempt, (5 - attempt)))
+            time.sleep(1)
+            if attempt >= 5:
+                raise
+            else:
+                continue
+        else:
```

Comment:
This means that if it's a different exception than `FlickrError` the function swallows the exception and returns an empty dict. That will be very hard to debug that exception, and an empty dict will make the other code not work properly. So remove this and let the original exception do its thing.

Reply:
Actually no because code under `else:` will only go if `try:` won't cause exception. So any non `FlickrError` will be raising. 
That was needed for exit from `for` loop if try was successful, but since you suggested `return photo.getSizes()` that `else:` is not needed any more